### PR TITLE
Remove deprecated JHipster Console (ELK)

### PR DESCRIPTION
**Pull request by PierreBesson**
As the JHipster Console has not been upgraded in over a year.  I'm removing the options from deployment subgens. I will change our documentations to replace them with generic instructions to connect JHipster apps to a generic a ELK setup.

---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
Really sad but I totally understand, as we work during our freetime, it's hard to maintain it for years :)

File: generators/generator-constants.js [hunk 1]
```diff
@@ -69,13 +69,7 @@ const DOCKER_KEYCLOAK = 'jboss/keycloak:11.0.1'; // The version should match the
...
 const DOCKER_KAFKA = `confluentinc/cp-kafka:${KAFKA_VERSION}`;
 const DOCKER_ZOOKEEPER = `confluentinc/cp-zookeeper:${KAFKA_VERSION}`;
-const DOCKER_SONAR = 'sonarqube:8.4.2-community';
-const DOCKER_JHIPSTER_CONSOLE = 'jhipster/jhipster-console:v4.1.0';
-const DOCKER_JHIPSTER_CURATOR = 'jhipster/jhipster-curator:v4.1.0';
-const DOCKER_JHIPSTER_ELASTICSEARCH = 'jhipster/jhipster-elasticsearch:v4.1.0';
-const DOCKER_JHIPSTER_LOGSTASH = 'jhipster/jhipster-logstash:v4.1.0';
-const DOCKER_JHIPSTER_IMPORT_DASHBOARDS = 'jhipster/jhipster-import-dashboards:v4.1.0';
-const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v4.1.0';
+const DOCKER_SONAR = 'sonarqube:8.4.1-community';
```
Comment (pascalgrimaud) [inline generators/generator-constants.js:72]: bad change, you downgrade the version of sonar here

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

**Comment by PierreBesson** [comment]
Not only too much work to maintain but also in retrospect I think it was a mistake to want to do everything as part of the JHipster project including providing our own deployments for the monitoring stack. Also, the world has changed a lot in the 5 years since releasing the Console and most people's focus has shifted to the Cloud and managed services. Also, it's not possible to guarantee good security with the current Console (no security patches, no encryption when forwarding logs, etc).

File: generators/generator-constants.js (hunk 1 above)
Comment (PierreBesson) [reply generators/generator-constants.js]: just fixed it

**Comment by pascalgrimaud** [comment]
you have some prettier format to do

**Comment by pascalgrimaud** [comment]
@PierreBesson : you're welcome, my friend ! :) it's team work !

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**